includes/admin/reporting/reports.php

```diff
@@ -277,14 +281,7 @@ function edd_register_overview_report( $reports ) {
 				'tile' => array(
 					'data_callback' => function () use ( $dates, $exclude_taxes, $currency ) {
 						$stats = new EDD\Stats();
-						return $stats->get_order_earnings( array(
-							'function'      => 'AVG',
-							'output'        => 'formatted',
-							'relative'      => true,
-							'range'         => $dates['range'],
-							'exclude_taxes' => $exclude_taxes,
-							'currency'      => $currency,
-						) );
+						return $stats->get_order_earnings();
```

Comment:
This doesn't seem right, moving towards just getting order earnings when we're actually trying to get the average order value for the period queried...

Reply:
This is producing this tile now:
<img width="354" alt="Screen Shot 2022-08-08 at 1 54 25 PM" src="https://user-images.githubusercontent.com/1390124/183512715-8c5a01af-30e1-4a50-8df3-ec3a49102200.png">

Instead of this:
<img width="339" alt="Screen Shot 2022-08-08 at 1 54 47 PM" src="https://user-images.githubusercontent.com/1390124/183512779-b481d254-e0fa-4477-9a69-5d1edf81c9cc.png">

Reply:
Whoops, maybe because I _deleted_ the array without replacing it. Fixed!